satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -3,27 +3,31 @@
 //--- bwagreement ---//
 
 model bwagreement (
-	key signature 
+	key signature
 
-	field signature blob 
+	field signature blob
 
 	field data blob
-	
+
```

Comment:
any reason for this empty line?

Reply:
no, I missed that when I was applying dbx formatter.